WcaOnRails/app/views/competitions/show_all_results.html.erb

```diff
@@ -1,17 +1,9 @@
 <% provide(:title, @competition.name) %>
 
 <%= render layout: 'results_nav' do %>
+  <%= tag :div, id: "selected-event", "data-selected": params[:event] %>
```

Comment:
This is the difference that affects caching: we're now sticking the eventId in the HTML outside of the `<% cache %>` block.

Before this PR, the first time someone visits this page with `&event=444bf`, it means that the cache value has 444bf baked into it, even if someone then visits it with `&event=clock`.

Now the information about which event is selected is outside the cached section, and the section within the `<% cache %>` is unaware of which event is selected.  Then javascript runs and displays the right event based on this line.

Reply:
Ahh, got it, thanks!